README.md

```diff
@@ -1 +1,56 @@
 # cartesi-paio
+
+
+## User transactions
+
+Users build an [EIP-712](https://eips.ethereum.org/EIPS/eip-712) signed transaction. A `SignedTransaction` consists of the pair `SigningMessage` and a `Signature`. They then submit it to a sequencer frontend.
+
+Note that this signed transaction contains the address of the destination dapp. It also contains a nonce (which is per application), and the maximum gas price this user is willing to pay _for DA_.
+
+
+## Sequencer frontend
```

Comment:
I don't know if I like the name "frontend"... but I can't come up with a better one.